### PR TITLE
refactored the whole book

### PR DESCRIPTION
**Pull request by joemoe**
did tons of stuff in the book.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,5 +5,4 @@ python:
 
 install: "pip install -q --use-mirrors Sphinx==1.1.3"
 
-script: sphinx-build -nW -b html -d _build/doctrees . _build/html
-
+script: sphinx-build -nW -b html -d _build/doctrees . _build/html
```
Comment (danrot) [inline .travis.yml]: The new line at the very end is very important :wink:

File: README.md [hunk 2]
```diff
@@ -1,2 +1,8 @@
-sulu-docs
-=========
+Sulu Documentation
+==================
+
+The [Sulu](http://sulu.io) Documentation can be found on (http://docs.sulu.io).
```
Comment (danrot) [inline README.md]: For direct links <> should be used: `<http://docs.sulu.io>`

File: book/creating-a-basic-website/about-the-sulu-content-architecture.rst [hunk 3]
```diff
@@ -1,6 +1,11 @@
 About the Sulu Content Architecture
 ===================================
 
+We already heard something about :doc:`../introduction/content-architecture`
+in the introduction. 
+Now we are starting to code will dig a little bit deeper.
+
+
```
Comment (danrot) [inline book/creating-a-basic-website/about-the-sulu-content-architecture.rst]: Is there a reason for two new lines here?

**Comment by danrot** [comment]
It looks like you are using hard tabs, but we have used soft tabs until now, can you please correct that in general?

File: book/creating-a-basic-website/creating-a-twig-template.rst [hunk 4]
```diff
@@ -0,0 +1,144 @@
+Creating a Twig Template
+========================
+
+Twig is an awesome option for rendering HTML. It got some nice features like
+blocks and inheritance. That's why we use and love Twig.
+
+.. note::
+	It is also possible to use plain PHP or any other template lannguage like Smarty,
+	but we strongly recommend Twig.
```
Comment (danrot) [inline book/creating-a-basic-website/creating-a-twig-template.rst]: Theoretically that should be possible, but I have never tested it. So I am not sure if it is a great idea to mention that here.

File: book/creating-a-basic-website/creating-a-twig-template.rst [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+
+Content
+^^^^^^^
+
+In the content everything you defined in your template is saved.
+If you got a title you could easily optain it from the content-var.
+
+.. code-block:: html
+
+	<title/>{{ content.title }}</title>
```
Comment (danrot) [inline book/creating-a-basic-website/creating-a-twig-template.rst]: I would maybe omit this line, since it is best practice to use the `sulu_seo` twig function for this.

File: book/creating-a-basic-website/creating-a-twig-template.rst [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+	...
+	<h1>{{ content.title }}</title>
+
+
+Extension
+^^^^^^^^^
+
+In the extension var Sulu writes content from Sulu extensions. Typically stuff
+that is defined in separate Tabs in the Sulu content section.
```
Comment (danrot) [inline book/creating-a-basic-website/creating-a-twig-template.rst]: Maybe it would be worth mentioning that these extensions are sticked to every single page, regardless of the template. That's what they are actually made for.

File: book/creating-a-basic-website/creating-a-twig-template.rst [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+At the moment there is the SEO and the excerpt extension, that could be used.
+
+Here is an example how it could look like in the backend. Notice the `Excerpt &
+Categories` tab next to the SEO tab.
```
Comment (danrot) [inline book/creating-a-basic-website/creating-a-twig-template.rst]: Maybe we should place texts within backticks in a single line. It seems to break the github higlighting otherwise :smile:

File: book/creating-a-basic-website/creating-a-twig-template.rst [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+
+.. figure:: ../../img/admin-extension-seo.png
+	:align: center
+
+You could include the SEO meta tags like this:
+
+.. code-block:: html
+
+	{{ sulu_meta_seo(extension, content) }}
```
Comment (danrot) [inline book/creating-a-basic-website/creating-a-twig-template.rst]: This method is deprecated, the `sulu_seo` method should be used instead, which does a little bit more than just rendering the meta tags (also renders the title, and sets the value correct for shadow pages as well).

File: book/creating-a-basic-website/creating-a-twig-template.rst [hunk 9]
```diff
@@ -0,0 +1,144 @@
...
+Navigation
+----------
+
+There is a Twig function that optains the menue.
+
+.. code-block:: html
+	:linenos:
+
+	<ul>
+		{% for item in sulu_navigation_root_tree('main') %}
```
Comment (danrot) [inline book/creating-a-basic-website/creating-a-twig-template.rst]: I think we should also explain what the `'main'` here is about.

File: book/creating-a-basic-website/creating-a-twig-template.rst [hunk 10]
```diff
@@ -0,0 +1,144 @@
...
+^^^^^^
+
+If there are images defined in your template you could render them by using this code:
+
+.. code-block:: html
+	:linenos:
+
+	{% for image in content.images %}
+	<div>
+		<img src="{{ image.thumbnails['200x100'] }}" alt="{{ image.name }}"/>
```
Comment (danrot) [inline book/creating-a-basic-website/creating-a-twig-template.rst]: Same as above, I think the link to image formats is missing (or how should people know which values can be used instead of `'200x100'`?)

File: book/creating-a-basic-website/creating-a-twig-template.rst [hunk 11]
```diff
@@ -0,0 +1,144 @@
...
+Default Template
+----------------
+
+Just have a look at our `default theme`_, that ships with our standard installation as long with our
+`default page templates`_ over at github.
+
+
+.. _default theme: https://github.com/sulu-io/sulu-standard/tree/develop/src/Client/Bundle/WebsiteBundle/Resources/themes/default
+.. _default page templates: https://github.com/sulu-io/sulu-standard/tree/develop/app/Resources/pages
+.. _example file: https://github.com/sulu-io/sulu-standard/blob/develop/src/Client/Bundle/WebsiteBundle/Resources/themes/default/templates/example.html.twig
```
Comment (danrot) [inline book/creating-a-basic-website/creating-a-twig-template.rst]: These links always link to the latest develop version of these files, which might cause troubles if we start publishing the documentation for different versions of Sulu. Don't know what the best idea to handle this is, but I just wanted to give a hint :smile:

File: book/creating-a-basic-website/index.rst [hunk 12]
```diff
@@ -1,13 +1,21 @@
...
     setup-a-webspace
-    adding-a-theme
     adding-a-template
+    adding-a-theme
+    creating-a-twig-template
     using-smart-content
     adding-localizations
 
+At first we'll have another look at the content architecture on a technical level
+and afterwards will create stuff.
```
Comment (danrot) [inline book/creating-a-basic-website/index.rst]: `we will create stuff`

File: book/getting-started/admin-interface.rst [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+--------
+
+In the settings section, stuff that is closer to the system is defined. We got user roles,
+categories, tags and the cache.
+
+
+User Roles
+^^^^^^^^^^
+
+There are several pre configured roles. The admin interface looks like this:
```
Comment (danrot) [inline book/getting-started/admin-interface.rst]: Actually there are no pre configured roles (depending on which `sulu:build` command you use there is maybe one admin role, but never several roles, unless you've created some on your own)

File: book/getting-started/admin-interface.rst [hunk 14]
```diff
@@ -0,0 +1,109 @@
...
+Sulu also got a tagging system.
+
+.. figure:: ../../img/admin-settings-tags.png
+	:align: center
+
+
+Snippets
+^^^^^^^^
+
+A Snippet is a global piece of contant, that could be used in every webspace.
```
Comment (danrot) [inline book/getting-started/admin-interface.rst]: `content`

File: book/getting-started/admin-interface.rst [hunk 15]
```diff
@@ -0,0 +1,109 @@
...
+There are several pre configured roles. The admin interface looks like this:
+
+.. figure:: ../../img/admin-settings-userroles.png
+	:align: center
+
+
+Categories
+^^^^^^^^^^
+
+Content could be added to categories.
```
Comment (danrot) [inline book/getting-started/admin-interface.rst]: Categories are not only for content, so I would describe this similar as you did for the tags.

File: book/getting-started/installation.rst [hunk 16]
```diff
@@ -87,8 +90,15 @@ should be fine for simple installations.
...
+:doc:`vhost`. Congratulations.
+
+
 .. _Jackalope Jackrabbit: https://github.com/jackalope/jackalope-jackrabbit
 .. _Jackalope Doctrine-Dbal: https://github.com/jackalope/jackalope-doctrine-dbal
 .. _Apache Jackrabbit: https://github.com/jackalope/jackalope-jackrabbit
 .. _Composer:  https://getcomposer.org/
 .. _MassiveBuildBundle: http://github.com/massiveart/MassiveBuildBundle
+
+
```
Comment (danrot) [inline book/getting-started/installation.rst]: One new line is enough :wink:

File: book/getting-started/setup.rst [hunk 17]
```diff
@@ -1,15 +1,20 @@
-Setup
-=====
+Setup a website!
+================
+
+On this page you'll learn how to configure your website. You'll define spaces and languages,
```
Comment (danrot) [inline book/getting-started/setup.rst]: I would write `webspaces`, just `spaces` confused me a bit.

File: book/getting-started/vhost.rst [hunk 18]
```diff
@@ -1,12 +1,20 @@
...
-==============
+Configure your local Webserver
+==============================
+
+You need to configure a virtual host for your webserver in order to run a clean Sulu instance.
+If you've already done so, skip this section.
+
+.. note::
+  It is a good decision to create a virtual host for each of your Sulu projects.
+  They are separated by domain and you'll got full control on what you expose.
```
Comment (danrot) [inline book/getting-started/vhost.rst]: For these notes we use four spaces at the beginning.

File: book/getting-started/vhost.rst [hunk 19]
```diff
@@ -99,3 +109,9 @@ Nginx
       }
   }
 
+
+.. warning::
+  Be sure to also configure your local host-file.
```
Comment (danrot) [inline book/getting-started/vhost.rst]: I would mention that this is only necessary if running Sulu locally.

Comment (danrot) [inline book/getting-started/vhost.rst]: And again four spaces.

File: book/index.rst [hunk 20]
```diff
@@ -1,5 +1,11 @@
 The Book
-========
+********
```
Comment (danrot) [inline book/index.rst]: Why did you change that? We always used `====` for h1 until now.

File: book/index.rst [hunk 21]
```diff
@@ -1,5 +1,11 @@
...
+
+When you work through the book you'll build a website based on Sulu from the scratch. At first we explain some of our very basic design decisions. Based on that we'll get your project started.
```
Comment (danrot) [inline book/index.rst]: We have a max line length of 80 characters.

File: book/introduction/backend-template.rst [hunk 22]
```diff
@@ -1,15 +1,29 @@
-Backend VS Templates
-====================
+Separation of Concerns
+======================
 
-As in any CMS Sulu completely separates content from design and allows
-template based content rendering. But due to the fact that usability, web
+As most of the modern Content Management Systems 
+Sulu completely separates content from design. It urges
+template based content rendering. Due to the fact that usability, web
```
Comment (danrot) [inline book/introduction/backend-template.rst]: I would suggest we always use as much of the 80 characters per line as possible, and don't split lines if not necessary.

File: book/introduction/content-architecture.rst [hunk 23]
```diff
@@ -1,4 +1,83 @@
...
+
+Once you created your webpsaces you could define languages. Pages could
+then be translated or mapped to another language as **shadow-** or **ghost-pages**.
+
+
+Page
+----
+
+As already heard in :doc:`backend-template` pages are configured in templates. They are created in
+webspaces and represent an entry in a certain menue. This pages are structured within your
```
Comment (danrot) [inline book/introduction/content-architecture.rst]: `These pages`, and I would also say the pages are contained within your webspace. The webspace itself does not really influence the structure of the page.

File: book/introduction/content-architecture.rst [hunk 24]
```diff
@@ -1,4 +1,83 @@
...
+webspace. Once you got your :doc:`../getting-started/setup` right, you could easily explore the
+Sulu default structure in the backend.
+
+
+Content-Type
+------------
+
+A template is build up of several Content-Types. A blogpost for example could exisit of the
```
Comment (danrot) [inline book/introduction/content-architecture.rst]: `exisit`is wrong for sure :wink: And I would use `consists`.

File: book/introduction/content-architecture.rst [hunk 25]
```diff
@@ -1,4 +1,83 @@
...
+
+
+Content-Type
+------------
+
+A template is build up of several Content-Types. A blogpost for example could exisit of the 
+following content-types:
+
+* Single line text (*Title*)
+* Multi line text (*Excerpt*)
```
Comment (danrot) [inline book/introduction/content-architecture.rst]: For the excerpt we would probably use the structure extension excerpt.

File: book/introduction/content-architecture.rst [hunk 26]
```diff
@@ -1,4 +1,83 @@
...
+* Richtext (*Content*)
+* Date (*Creation-Date*)
```
Comment (danrot) [inline book/introduction/content-architecture.rst]: FYI: Sulu already remembers some dates by default, and passes them to the twig template. Such a field would be necessary if you want to set the publish date on your own, but we also plan to build that into the core of Sulu.

File: book/introduction/content-architecture.rst [hunk 27]
```diff
@@ -1,4 +1,83 @@
...
+* List of Tags (*Tags*)
+
+A detailed list of all the content-types is covered in :doc:`../creating-a-basic-website/adding-a-template`.
+
+
+Assets
+------
+
+Assets are media files like Images and Downloads. They are shared through all the Websites on the Instance.
```
Comment (danrot) [inline book/introduction/content-architecture.rst]: Should `Images`, `Downloads`, `Websites` and `Instance` really be capitalized.

File: book/introduction/content-architecture.rst [hunk 28]
```diff
@@ -1,4 +1,83 @@
...
+
+
+Snippets
+--------
+
+Snippets are very simelar to assets. They are small pieces of content that could be shown on
```
Comment (danrot) [inline book/introduction/content-architecture.rst]: `similar`

File: book/introduction/content-architecture.rst [hunk 29]
```diff
@@ -1,4 +1,83 @@
...
+--------
+
+Snippets are very simelar to assets. They are small pieces of content that could be shown on 
+several pages in several webspaces.
+
+
+Contacts
+--------
+
+Contacts reperesent personal information. It is a very generic type of as Sulu system users are also
```
Comment (danrot) [inline book/introduction/content-architecture.rst]: `represent`

Comment (danrot) [inline book/introduction/content-architecture.rst]: And the I am not sure what the second sentence should exactly mean, but it doesn't sound grammatically right.

File: book/introduction/content-architecture.rst [hunk 30]
```diff
@@ -1,4 +1,83 @@
...
+represented through Contacts.
+
+
+Whatever you want
+-----------------
+
+Sulu is very extensible. You got existing content you want to integrate through DBAL? Do it.
+An API that delivers content? Integrate it. Other apps you've already coded. Use them.
+
+A more detailled documentation could be found in the secion :doc:`../creating-a-basic-website/about-the-sulu-content-architecture`.
```
Comment (danrot) [inline book/introduction/content-architecture.rst]: `section`

File: book/introduction/content-architecture.rst [hunk 31]
```diff
@@ -1,4 +1,83 @@
...
+
+Contacts reperesent personal information. It is a very generic type of as Sulu system users are also
+represented through Contacts.
+
+
+Whatever you want
+-----------------
+
+Sulu is very extensible. You got existing content you want to integrate through DBAL? Do it.
+An API that delivers content? Integrate it. Other apps you've already coded. Use them.
```
Comment (danrot) [inline book/introduction/content-architecture.rst]: `Other apps you've already coded?`

File: book/introduction/content-architecture.rst [hunk 32]
```diff
@@ -1,4 +1,83 @@
...
+
+Snippets are very simelar to assets. They are small pieces of content that could be shown on 
+several pages in several webspaces.
+
+
+Contacts
+--------
+
+Contacts reperesent personal information. It is a very generic type of as Sulu system users are also
+represented through Contacts.
```
Comment (danrot) [inline book/introduction/content-architecture.rst]: Same as above, should `Contacts` be capitalized?

File: book/introduction/index.rst [hunk 33]
```diff
@@ -1,11 +1,23 @@
...
+    right-choice
     content-management-platform
+    sulu-focus
+    structure
     components
     backend-template
     content-architecture
 
+It is important to understand the underlaying concepts of Sulu,
+but if you wan't to skip this part and start coding, head over
```
Comment (danrot) [inline book/introduction/index.rst]: `want`

File: book/introduction/right-choice.rst [hunk 34]
```diff
@@ -0,0 +1,26 @@
...
+
+* Do you want to work with the Symfony stack?
+* You got a running app based on Symfony and want to add a content management layer?
+* Do you need a clean, solid development setup?
+* Default solutions won't fit the requirements?
+* You want to create a mondern website based on modern technology?
+* Do you need total freedom over your frontend code?
+* Does your project require multiple languages?
+* Should there be several portals?
+* Usability and desgin of the backend are important?
```
Comment (danrot) [inline book/introduction/right-choice.rst]: `design`

File: book/introduction/structure.rst [hunk 35]
```diff
@@ -0,0 +1,27 @@
...
+
+.. image:: ../../img/system-architecture.png
+	:align: center
+	:alt: Sulu System Architecture
+
+
+Data
+----
+
+Symfony uses PHPCR, Doctrine and MySQL, MariaDB or Jackrabbitt.
```
Comment (danrot) [inline book/introduction/structure.rst]: `Jackrabbit`

File: book/introduction/sulu-focus.rst [hunk 36]
```diff
@@ -0,0 +1,37 @@
+On what does Sulu focus?
+========================
+
+While developing Sulu we had a special vision in mind. Besides "Content management made awesome"
+we wanted to create a technology that
+
+* supports what a business needs.
+* looks good and is easy to use.
+* helps developers excell.
```
Comment (danrot) [inline book/introduction/sulu-focus.rst]: `excel`

File: bundles/index.rst [hunk 37]
```diff
@@ -1,13 +1,16 @@
 Bundles
 =======
 
-Sulu's functionality is integrated into the Symfony framework using several
-bundles.
+The Sulu code is structured in Symfony Bundles. Some of them are more important to developers who work with Sulu than others. In this documentation we documented the most important ones.
```
Comment (danrot) [inline bundles/index.rst]: 80 character per line

File: bundles/index.rst [hunk 38]
```diff
@@ -1,13 +1,16 @@
...
+
+**We documented the following bundles**
 
 .. toctree::
      :maxdepth: 2
 
-     admin/index
+     admin/index.rst
```
Comment (danrot) [inline bundles/index.rst]: Is the `.rst` at the end necessary?

File: conf.py [hunk 39]
```diff
@@ -263,3 +263,11 @@
...
 # If true, do not generate a @detailmenu in the "Top" node's menu.
 #texinfo_no_detailmenu = False
+
+import os
+on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
+
+if not on_rtd:  # only import and set the theme if we're building docs locally
+    import sphinx_rtd_theme
+    html_theme = 'sphinx_rtd_theme'
+    html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
```
Comment (danrot) [inline conf.py]: This part seems to break the build :wink:

File: cookbook/index.rst [hunk 40]
```diff
@@ -1,7 +1,9 @@
 Cookbook
 ========
 
-Awesome recipies for lunch
+Here are some awesome recepies for lunch. The Cookbook covers some advanced topics which each covers a very special problem. Create your very own menu with our recepies.
```
Comment (danrot) [inline cookbook/index.rst]: 80 chars

File: cookbook/index.rst [hunk 41]
```diff
@@ -16,3 +18,5 @@ Awesome recipies for lunch
     integrating-custom-filters
     extend-entities
     using-elasticsearch
+
+It is possible to work through the recepies, altought most of the people will pick the ones, which are most similar to their own tasks. We're open to `suggestions <http://sulu.io/en/contact>`_.
```
Comment (danrot) [inline cookbook/index.rst]: 80 chars, and if the syntax with the `_` at the end is used, the link has to be defined at the end, shouldn't it?

**Comment by danrot** [comment]
Can you please check the line length in general? Our rule is that it should not exceed 80 characters.

File: .travis.yml (hunk 1 above)
Comment (joemoe) [inline .travis.yml]: don't know why I removed it! tried to do stuff.

File: README.md (hunk 2 above)
Comment (joemoe) [inline README.md]: Switched to that afterwards, will check it.

Comment (joemoe) [inline README.md]: ha, i know why i did this. because it's md not restructured text. it's just for github.

File: book/creating-a-basic-website/about-the-sulu-content-architecture.rst (hunk 3 above)
Comment (joemoe) [inline book/creating-a-basic-website/about-the-sulu-content-architecture.rst]: yes. i want to cross reference and we said we wont to have an introduction and short summary or outview on every page.

File: book/creating-a-basic-website/creating-a-twig-template.rst (hunk 4 above)
Comment (joemoe) [inline book/creating-a-basic-website/creating-a-twig-template.rst]: Removed

File: book/creating-a-basic-website/creating-a-twig-template.rst (hunk 5 above)
Comment (joemoe) [inline book/creating-a-basic-website/creating-a-twig-template.rst]: done.

File: book/creating-a-basic-website/creating-a-twig-template.rst (hunk 6 above)
Comment (joemoe) [inline book/creating-a-basic-website/creating-a-twig-template.rst]: good point.

File: book/creating-a-basic-website/creating-a-twig-template.rst (hunk 7 above)
Comment (joemoe) [inline book/creating-a-basic-website/creating-a-twig-template.rst]: ok.

File: book/creating-a-basic-website/creating-a-twig-template.rst (hunk 8 above)
Comment (joemoe) [inline book/creating-a-basic-website/creating-a-twig-template.rst]: so we should also adopt the example template files. creating a ticket.
https://github.com/sulu-io/sulu-standard/issues/504

File: README.md (hunk 2 above)
Comment (danrot) [inline README.md]: Alright :-)

File: book/creating-a-basic-website/creating-a-twig-template.rst (hunk 8 above)
Comment (danrot) [inline book/creating-a-basic-website/creating-a-twig-template.rst]: Definitely

File: book/creating-a-basic-website/creating-a-twig-template.rst (hunk 10 above)
Comment (joemoe) [inline book/creating-a-basic-website/creating-a-twig-template.rst]: added a part about the image_formats.xml

File: book/creating-a-basic-website/creating-a-twig-template.rst (hunk 11 above)
Comment (joemoe) [inline book/creating-a-basic-website/creating-a-twig-template.rst]: i'll point them to master. have to look what others do.

File: book/creating-a-basic-website/index.rst (hunk 12 above)
Comment (joemoe) [inline book/creating-a-basic-website/index.rst]: your right.

File: book/getting-started/admin-interface.rst (hunk 13 above)
Comment (joemoe) [inline book/getting-started/admin-interface.rst]: ok

File: book/getting-started/installation.rst (hunk 16 above)
Comment (joemoe) [inline book/getting-started/installation.rst]: i like new lines.

File: book/getting-started/setup.rst (hunk 17 above)
Comment (joemoe) [inline book/getting-started/setup.rst]: ok.

File: book/getting-started/vhost.rst (hunk 18 above)
Comment (joemoe) [inline book/getting-started/vhost.rst]: ok.

File: book/getting-started/vhost.rst (hunk 19 above)
Comment (joemoe) [inline book/getting-started/vhost.rst]: ok.

File: book/index.rst (hunk 20 above)
Comment (joemoe) [inline book/index.rst]: i looked it up and made a structure, but rst seems to just dont give a ...

File: book/index.rst (hunk 21 above)
Comment (joemoe) [inline book/index.rst]: changed that almost everywhere

File: book/introduction/backend-template.rst (hunk 22 above)
Comment (joemoe) [inline book/introduction/backend-template.rst]: did so.

File: book/index.rst (hunk 21 above)
Comment (danrot) [inline book/index.rst]: Almost? :smile:

File: book/introduction/content-architecture.rst (hunk 23 above)
Comment (joemoe) [inline book/introduction/content-architecture.rst]: ok.

File: book/index.rst (hunk 20 above)
Comment (danrot) [inline book/index.rst]: Right, RST implies the priority of the first appearance, but the [Symfony Documentation standard](http://symfony.com/doc/current/contributing/documentation/standards.html) has some rules :-)

File: book/introduction/content-architecture.rst (hunk 25 above)
Comment (joemoe) [inline book/introduction/content-architecture.rst]: ok. made something different up.

File: book/introduction/content-architecture.rst (hunk 26 above)
Comment (joemoe) [inline book/introduction/content-architecture.rst]: hmhm. not that creative. :)

File: book/introduction/content-architecture.rst (hunk 27 above)
Comment (joemoe) [inline book/introduction/content-architecture.rst]: no.

File: book/introduction/content-architecture.rst (hunk 29 above)
Comment (joemoe) [inline book/introduction/content-architecture.rst]: you're right.

File: book/introduction/content-architecture.rst (hunk 31 above)
Comment (joemoe) [inline book/introduction/content-architecture.rst]: changed it a little bit.

File: book/introduction/index.rst (hunk 33 above)
Comment (joemoe) [inline book/introduction/index.rst]: that's a good one.

File: book/introduction/right-choice.rst (hunk 34 above)
Comment (joemoe) [inline book/introduction/right-choice.rst]: .

File: book/introduction/sulu-focus.rst (hunk 36 above)
Comment (joemoe) [inline book/introduction/sulu-focus.rst]: :+1:

File: bundles/index.rst (hunk 37 above)
Comment (joemoe) [inline bundles/index.rst]: :+1:

File: bundles/index.rst (hunk 38 above)
Comment (joemoe) [inline bundles/index.rst]: no. don't know why i added it.

File: conf.py (hunk 39 above)
Comment (joemoe) [inline conf.py]: hm. i'll remove it before pushing.

File: cookbook/index.rst (hunk 40 above)
Comment (joemoe) [inline cookbook/index.rst]: :+1:

File: cookbook/index.rst (hunk 41 above)
Comment (joemoe) [inline cookbook/index.rst]: no works both.
